sys-cluster/charliecloud/charliecloud-0.9.10.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+if [[ ${PV} == "9999" ]] ; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/hpc/${PN}.git"
+	S="${WORKDIR}/${P}"
+else
+	SRC_URI="https://github.com/hpc/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+fi
+
+DESCRIPTION="Lightweight user-defined software stacks for high-performance computing"
+HOMEPAGE="https://hpc.github.io/charliecloud/"
+
+SLOT="0"
+LICENSE="Apache-2.0"
+IUSE="doc examples +pv"
+
+RDEPEND=""
+DEPEND="${RDEPEND}
+	pv? ( sys-apps/pv )
+	doc? (
+		dev-python/sphinx
+		dev-python/sphinx_rtd_theme
+		net-misc/rsync
+	)"
+
+src_compile() {
+	emake
+	use doc && emake -C doc-src
+}
+
+src_install() {
+	emake install PREFIX="${EPREFIX}/usr" DESTDIR="${D}" LIBEXEC_DIR="libexec/${PF}"
+	if use doc; then
+		local HTML_DOCS=(doc/.)
+	fi
+	if use examples; then
+		docompress -x "/usr/share/doc/${PF}/examples"
+		dodoc -r examples
+	fi
+	rm -rf "${ED}/usr/share/doc/charliecloud" || die
+	einstalldocs
+}
+
```

Comment:
repoman complains about this empty line.

Reply:
Thanks. I was not clever enough to re-run repoman after squashing, since I usually use it just for comitting. Lesson learnt :wink:.